frame/ddc-metrics-offchain-worker/src/lib.rs

```diff
@@ -274,7 +274,10 @@ impl<T: Config> Module<T> where <T as frame_system::Config>::AccountId: AsRef<[u
         let s_next_at = StorageValueRef::persistent(b"ddc-metrics-offchain-worker::next-at");
 
         match s_next_at.mutate(|current_next_at| {
-            let current_next_at = current_next_at.unwrap_or(Some(T::BlockNumber::default()));
+            let current_next_at = match current_next_at {
+                Ok(Some(val)) => Some(val),
+                _ => Some(T::BlockNumber::default()),
+            };
```

Comment:
The issue is cause by current_next_at [type changing](https://github.com/paritytech/substrate/pull/9119).
`unwrap_or` behaves differently on `Option` and `Result`. Preveiosly, called on `Option` it has been returning `Some` or default `Some` in parentheses. After refactoring called on `Result` it never return the default value. Insted, before the fix it has been returning `None` what trigger [else case](https://github.com/Cerebellum-Network/pos-network-node/pull/153/commits/191a12caa3a4c2fd9a3dfb93d868f4647b8fd27f#diff-2cf146c84dbe7f3d47b44a1345d0bed15228bcd3aecb643c6c0abd42f7803a80R299) the error.